docs/tutorial-modifier-roles/add-role.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+# Add Role
+
+After confirming the transaction, you should return to the Zodiac App, where your configured Role Modifier will be displayed in the left side panel. Click anywhere on the modifier's card to access the Roles App.
+
+![Roles App - zero state](/img/tutorial/roles_03.png)
+
+## Create a Role
+
+![Roles App - zero state](/img/tutorial/roles_04.png)
+Click on the "Create a Role" button to open the Role view where you can add members and targets to the role. The Role name is created automatically.
+
+Once you've added the appropriate members and targets to this role (see below), you can click "Create Role". Only then will your changes to the role be saved. Note that you must connect your wallet (and be a key signer in connected safe) in order to create the role.
+![Create Role view](/img/tutorial/roles_05.png)
+
+## Members
+
+![Roles App - Add a Member modal](/img/tutorial/roles_06.png)
+
+Members are accounts that that the role is assigned to. Once an account is assigned the role, it can call the targets on behalf of the avatar by passing transactions to the avatar for execution (via the Roles Modifier).
```

Comment:
```suggestion
Members are accounts that the role is assigned to. Once an account is assigned the role, it can call the targets on behalf of the avatar by passing transactions to the avatar for execution (via the Roles Modifier).
```

Reply:
> Also, for some reason images are not showing up... 🤔

I believe this is because for github, the image links need to be the absolute links vs relative. But they do work in the docusaurus environment, at least for me:

<img width="1482" alt="image" src="https://user-images.githubusercontent.com/80348806/156892668-28c15183-35cd-4531-914d-dbcce40e89ad.png">